src/pages/billing/CreateGCPBillingProject.ts

```diff
@@ -8,13 +8,24 @@ import Events from 'src/libs/events'
 import { formHint, FormLabel } from 'src/libs/forms'
 import * as Utils from 'src/libs/utils'
 import { billingProjectNameValidator } from 'src/pages/billing/List'
+import { BillingAccount } from 'src/pages/billing/models/BillingAccount'
 import validate from 'validate.js'
 
 
+interface CreateGCPBillingProjectProps {
+  billingAccounts: Record<string, BillingAccount>
+  chosenBillingAccount?: BillingAccount
+  setChosenBillingAccount: (BillingAccount) => void
+  billingProjectName?: string
+  setBillingProjectName: (string) => void
+  existing: any
+  disabled?: boolean
+}
+
 const CreateGCPBillingProject = ({
```

Comment:
I think I am missing something.... where does GCPBillingProjectWizard get called up from?

Reply:
src/pages/billing/List.js
line 444 (in this PR - it'll be 430 in a version that excludes the changes for the azure component)